metricbeat/module/kubernetes/state_node/_meta/data.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@timestamp": "2017-05-04T17:47:20.829Z",
+  "beat": {
+    "hostname": "X1",
+    "name": "X1",
+    "version": "6.0.0-alpha1"
+  },
+  "kubernetes": {
+    "node": {
+      "allocatable": {
```

Comment:
This sounds good to me, would it be possible to use `node.state` as metricset name?